components/brave_ads/core/internal/settings/settings_unittest_util.h

```diff
@@ -15,6 +15,9 @@ void OptOutOfBraveNewsAds();
 void OptOutOfNewTabPageAds();
 
 void OptOutOfNotificationAds();
+
+void OptOutOfSearchResultAds();
```

Comment:
nit: Move to after `OptOutOfNotificationAds` and `SetMaximumNotificationAdsPerHour` because those are related.